base/src/main/java/io/spine/util/Preconditions2.java

```diff
@@ -54,9 +54,31 @@ private Preconditions2() {
      */
     @CanIgnoreReturnValue
     public static String checkNotEmptyOrBlank(String str) {
```

Comment:
Why don't we reuse the overloaded `checkNotEmptyOrBlank(str, errorMessage)` here?

Reply:
To save on formatting or array allocation for vararg if all goes well.